chainstate_test_framework/src/framework.rs

```diff
@@ -42,6 +39,14 @@ pub struct TestFramework {
 }
 
 impl TestFramework {
+    /// Creates a new test framework
+    pub fn new(chainstate: TestChainstate, block_indexes: Vec<BlockIndex>) -> Self {
```

Comment:
As far as I can see, this construct isn't used. Am I missing something?

Reply:
Removed